sphinx/builders/latex.py

```diff
@@ -247,7 +250,7 @@ def validate_config_values(app):
             app.warn('latex_use_parts conflicts with latex_toplevel_sectioning, ignored.')
         else:
             app.warn('latex_use_parts is deprecated. Use latex_toplevel_sectioning instead.')
-            app.config.latex_toplevel_sectioning = 'parts'  # type: ignore
+            app.config.latex_toplevel_sectioning = 'part'  # type: ignore
```

Comment:
Ah, sorry. This is not a part of this pull request.
It was already commited into stable branch: https://github.com/sphinx-doc/sphinx/commit/12096b3b863f40d10fce94e03abc4b8d3f856eb1.